docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,350 @@
+
+  To enable Redis cluster mode in Teleport add `mode=cluster` parameter to connection uri in teleport config file.
+
+  ```yaml
+  databases:
+    - name: "redis-cluster"
+      uri: "rediss://redis.example.com:6379?mode=cluster"
+  ```
+
+</Admonition>
+
+## Step 5/5. Connect
+
+Log into your Teleport cluster and see available databases:
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+$ tsh db ls
+# Name          Description     Labels
+# ------------- --------------- --------
+# example-redis Example Redis env=dev
+```
+
+To connect to a particular database instance, first retrieve its certificate
+using `tsh db login` command:
+
+```code
+$ tsh db login example-redis
+```
+
+<Admonition type="tip" title="Tip">
+  You can be logged into multiple databases simultaneously.
+</Admonition>
+
+You can optionally specify the database name and the user to use by default
+when connecting to the database instance:
+
+```code
+$ tsh db login --db-user=alice example-redis
+```
+
+<Admonition type="note" title="Note">
+  If flag `--db-user` is not provided Teleport logs in as the `default` user.
+</Admonition>
+
+Once logged in, connect to the database:
+
+```code
+$ tsh db connect example-redis
+```
+
+<Admonition type="note" title="Note">
+  The `redis-cli` command-line client should be available in the system `PATH` in order to be
+  able to connect.
+</Admonition>
+
+Now you can log in as the previously created user using the below command:
+
+```
+AUTH alice STRONG_GENERATED_PASSWORD
+```
+
+To log out of the database and remove credentials:
+
+```code
+# Remove credentials for a particular database instance.
+$ tsh db logout example-redis
+# Remove credentials for all database instances.
+$ tsh db logout
+```
+
+### Supported Redis commands
+
+  <Tabs>
+    <TabItem label="Standalone">
+
+      Redis in standalone mode doesn't support below commands.
+
+      - HELLO
+      - PUNSUBSCRIBE
+      - SSUBSCRIBE
+      - SUNSUBSCRIBE
+
+      If one of the listed commands is called Teleport returns `ERR Teleport: not supported by Teleport` error.
+
+    </TabItem>
+    <TabItem label="Cluster">
+
+      Redis in cluster mode doesn't support below commands:
+
+      - ACL
+      - ASKING
+      - CLIENT
+      - CLUSTER
+      - CONFIG
+      - DEBUG
+      - EXEC
+      - HELLO
+      - INFO
+      - LATENCY
+      - MEMORY
+      - MIGRATE
+      - MODULE
+      - MONITOR
+      - MULTI
+      - PFDEBUG
+      - PFSELFTEST
+      - PSUBSCRIBE
+      - PSYNC
+      - PUNSUBSCRIBE
+      - PUNSUBSCRIBE
+      - READONLY
+      - READWRITE
+      - REPLCONF
+      - REPLICAOF
+      - ROLE
+      - SCAN
+      - SCRIPT DEBUG
+      - SCRIPT KILL
+      - SHUTDOWN
+      - SLAVEOF
+      - SLOWLOG
+      - SSUBSCRIBE
+      - SUNSUBSCRIBE
+      - SYNC
+      - TIME
+      - WAIT
+      - WATCH
+
+      If one of the listed commands above is called Teleport returns `ERR Teleport: command not supported` error.
```

Comment:
Since this is a long list, we should put this sentence before it so readers have all the context and don't accidentally skip this information.

Reply:
Done